frontend/src/modules/assetManagement/templates/assetManagementModalNewAsset.hbs

```diff
@@ -38,6 +38,46 @@
             <textarea class="asset-description required" name="description" placeholder="{{t 'app.assetdescription'}}">{{ description }}</textarea>
         </div>
 
+        <div class="form-group">
+            <label for="attribution">
+                {{t 'app.assetattribution'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <textarea class="asset-attribution required" name="attribution" placeholder="{{t 'app.assetattribution'}}">{{ attribution }}</textarea>
+        </div>
+
+        <div class="form-group">
+            <label for="source">
+                {{t 'app.assetsource'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <textarea class="asset-source required" name="source" placeholder="{{t 'app.assetsource'}}">{{ source }}</textarea>
+        </div>
+
+        <div class="form-group">
+            <label for="licence">
+                {{t 'app.assetlicence'}} <span class="req">*</span>
+                <span class="error"></span>
+            </label>
+            <select class="asset-licence required" name="licence" >
+                <option value=""></option>
```

Comment:
If we use select we can't have "custom" license types.

Reply:
Would there be a better way to define a set of values someone can customise that can be loaded here? Something in properties.schema?

Reply:
Not really as far as I know. Maybe leave the dropdown but also add a text-input to allow custom text or only use the custom text-input.